internal/cli/cmd/cli.go

```diff
@@ -64,6 +65,13 @@ const (
 	cliName = "kbcli"
 )
 
+// TODO: add more commands
+var whiteList = map[string]bool{
+	"logout":  true,
+	"cluster": true,
+	"addon":   true,
```

Comment:
Why `cluster` and `addon` should login first?

Reply:
Sorry, here are the commands I randomly added first, and I will change them later as needed.

Reply:
ACK